### DISCUSSION TITLE
nwg-shell for second user

### DISCUSSION THREAD

**Discussion by vrs804** [nwg-piotr/nwg-shell#466] [Q&A] [answered]
I have an nwg-iso, recently created a second user and I need to enable the user interface. Would this be with the terminal and the command nwg-shell-installer -a ? thanks

**Comment by nwg-piotr** [comment, marked as answer]
Funny thing that I've never tried :) Yes, `nwg-shell-installer -a` should copy all configs to their destination.

**Comment by vrs804** [comment]
it's right nwg-shell -installer -a -hypr ( I also installed HYP, but for me sway is enoug )  It is the solution
Merry christmas

**Comment by nwg-piotr** [comment]
Marry Christmas to you, too.
